features/command_line_interface/minimum_score.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Brake if overall score is bellow minimum
+  In order to breake the Continuous Integration builds based on a score threshold
+  Rubycritic returns the exit status according with the score
+
+  Scenario: Status indicates an error when score below the minimum
+    Given the smelly file 'smelly.rb'
+    When I run rubycritic --minimum-score 100 smelly.rb
+    Then the exit status indicates an error
+  Scenario: Status indicates a success when score is above the minimum
+    Given the smelly file 'smelly.rb'
```

Comment:
Same here - I, as the user have **no** idea what the real score is. So that we're using "50", "100" and so on is confusing to me. Let's put all of those numbers in the step definition so that it's obvious.